dist/src/main/resources/api/backup-management-api.yaml

```diff
@@ -109,6 +110,77 @@ paths:
           description: Request from gateway to broker timed out
           $ref: '#/components/responses/Error'
 
+  # Section on historic backups
+  /backup-history:
+    post:
+      summary: Takes a historic backup
+      description: Start taking a backup with the given backupId
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TakeBackupHistoryRequest'
+      responses:
+        '202':
+          description: A backup has been successfully scheduled
+          $ref: '#/components/responses/TakeBackupHistoryResponse'
+        '400':
+          description: In case something is wrong with backupId, e.g. the same backup ID already exists.
+        '500':
+          description: All other errors, e.g. ES returned error response when attempting to create a snapshot.
+        '502':
+          description: Elasticsearch is not accessible, the request can be retried when it is back.
+    get:
+      summary: Lists all available historic backups
+      description: |
+        Returns a list of all available backups with their state and additional info,
+        sorted in descending order of backupId.
+      responses:
+        '200':
+          description: OK
+          $ref: '#/components/responses/HistoryBackupList'
+        '404':
+          description: Backup repository is not configured.
+        '400':
+          description: In case something is wrong with backupId, e.g. the same backup ID already exists.
```

Comment:
:x: There should be no 400 returned here or at the very least the description is wrong because here we don't provide a backup id.

Reply:
:+1: